src/weights.jl

```diff
@@ -628,25 +562,23 @@ function quantile(v::RealVector{V}, w::AbstractWeights{W}, p::RealVector) where
     return out
 end
 
-# similarly to statistics.jl in Base
+# similar function in Base statistics.jl
 function bound_quantiles(qs::AbstractVector{T}) where T<:Real
```

Comment:
Somewhat unrelated, but this function has been removed from Base. We should probably remove it too.